test/extensions/filters/network/zookeeper_proxy/filter_test.cc

```diff
@@ -30,7 +31,19 @@ class ZooKeeperFilterTest : public testing::Test {
   ZooKeeperFilterTest() { ENVOY_LOG_MISC(info, "test"); }
 
   void initialize() {
-    config_ = std::make_shared<ZooKeeperFilterConfig>(stat_prefix_, 1048576, scope_);
+    // Populate latency threshold for Multi opcode. The latency thresholds of all other opcode
+    // fallback to the default threshold. Since we did not specify the default threshold, it
+    // fallbacks to 100 millisecond.
```

Comment:
It's implicit for testing the default behavior. could we have 2 new tests?

like
```
config =  std::make_shared<ZooKeeperFilterConfig>(stat_prefix_, 1048576, latency_thresholds_, scope_);
EXPECT_EQ(config.errorbudgetDecision(opcode, 99), FAST);
EXPECT_EQ(config.errorbudgetDecision(opcode, 101), SLOW);
```
one test have default value, another doesn't

Reply:
Thanks, just add more tests in both `config_test.cc` and `filter_test.cc` to test below four config scenario:
1. No latency thresholds specified
2. Only default latency threshold specified
3. Only non-default latency threshold specified
4. Both default and non-default latency thresholds specified